### ISSUE TITLE
ANSI escape/color codes do not work in the run window.

### ISSUE THREAD

**Issue by Directory** [intellij-rust/intellij-rust#9053]
## Environment

* **IntelliJ Rust plugin version: Latest**
* **1.64 nightly**
* **intellij IDEA 2022.1.3**
* **Windows 10.**

When running any rust program (where it internally calls `cargo run`) the build output (in the build tab) will have color but the actual stdout from the rust program itself (when calling println!() or something) will not have colors or other ansi escape codes applied. Here is a relevant [issue](https://github.com/mackwic/colored/issues/106) with a popular print coloring crate that wraps strings in ansi codes for a certain color. As you can see in that issue and from my experience, colors appear fine when executing the program manually in another terminal and even in the intellij built in terminal but not in the actual "run" output window.

Another common place I've ran into this issue is with the rocket.rs library. It by default adds fancy colors for each logged http request but ive never had those colors display to me in the Run window. Ive only seen the colors either when running an http server window in the Terminal window or through ssh on a linux server. 

To reproduce just `println!()` any hello world string wrapped in some ansi color code. And watch as no color is applied to the text in the Run window but will in most other terminals.

**Comment by LoganDark** [comment]
I don't think this issue can be solved by IntelliJ-Rust. JetBrains for some reason decided that PyCharm is the only IDE that should run a proper terminal emulator in the Run window. Why they think a high-level interpreted language deserves it more than C/C++/Rust does is completely beyond me.

**Comment by Directory** [comment]
i figured out a partial solution/workaround
toggle "emulate terminal in output console".
this however comes at the cost of kind of buggy build output
![image](https://user-images.githubusercontent.com/26686708/182831127-e029ca71-8125-43a6-9109-09c07dd95c05.png)

**Comment by LoganDark** [comment]
Wow, apparently right after I made that comment they went and implemented terminal in the run window for Rust. Is there a way to turn that on globally for all launch configurations forever? I hate the default run window.

**Comment by mchernyavsky** [comment]
@LoganDark Now you can enable `org.rust.cargo.emulate.terminal` in `Help | Find Action... | Registry...`. It enables `Emulate terminal in output console` in all newly created configurations by default.

**Comment by LoganDark** [comment]
> @LoganDark Now you can enable `org.rust.cargo.emulate.terminal` in `Help | Find Action... | Registry...`. It enables `Emulate terminal in output console` in all newly created configurations by default.

Thank you!
